types/jasmine/index.d.ts

```diff
@@ -324,8 +320,6 @@ declare namespace jasmine {
 
     function stringMatching(str: string | RegExp): AsymmetricMatcher<string>;
 
-    function formatErrorMsg(domain: string, usage: string): (msg: string) => string;
```

Comment:
I just double-checked all the deletions and found that this function and `Suite#getFullName` still exist. They're private APIs but it's possible that user code could be using them, at least in the case of `formatErrorMessage`. (I'm not sure it's even possible to get a reference to a `Suite` from outside Jasmine. Should I put those two methods back?

Reply:
If they're private APIs and you haven't documented them in the official jasmine docs, I'm ok with deleting them.

Reply:
It is possible to get the suites and specs from `jasmine.env.topSuite()` and it's children.

Reply:
Why not just add `@deprecated`? You can also extract all those methods into `legacy.d.ts` and incorporate in the module export (so you can phase those out incrementally)